sw/device/lib/runtime/BUILD

```diff
@@ -0,0 +1,98 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+package(default_visibility = ["//visibility:public"])
+
+load("//rules:opentitan.bzl", "OPENTITAN_CPU")
+
+cc_library(
+    name = "epmp",
+    srcs = ["epmp.c"],
+    hdrs = ["epmp.h"],
+    target_compatible_with = [OPENTITAN_CPU],
+    deps = [
+        "//sw/device/lib/base/freestanding",
```

Comment:
I think https://github.com/bazelbuild/buildtools would have reformatted this. Can we make sure to have build file formatting running in CI or something?

Reply:
buildifier added to the workspace and it's on the todo list to add it to the CI. I try to run it before committing, but sometimes I miss it. 
Running buildifier on everything doesn't affect this file. Is there something specific?

Reply:
I think I just imageined it. =)